arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1100,57 +1045,57 @@
       }
     },
     "StorageEndpointProperties": {
-      "description": "Contains the user facing view of the properties of the Storage Endpoint provided by the customer for file upload",
+      "description": "The properties of the Storage Endpoint for file upload.",
       "type": "object",
       "properties": {
         "sasTtlAsIso8601": {
-          "description": "SAS time to live",
-          "format": "date-time",
+          "description": "SAS time to live. Range: 1 Min - 1 Day.",
           "type": "string"
         },
         "connectionString": {
-          "description": "The account key credentials for storage account selected by customer for uploading files",
+          "description": "The account key credentials for storage account selected by customer for uploading files.",
           "type": "string"
         },
         "containerName": {
-          "description": "The root container name where all files will be uploaded",
+          "description": "The root container name where all files will be uploaded.",
           "type": "string"
         }
       }
     },
     "MessagingEndpointProperties": {
-      "description": "Contains the user facing view of the properties of the Messaging Endpoints used by this IoT Hub",
+      "description": "The properties of the Messaging Endpoints used by this IoT Hub.",
       "type": "object",
       "properties": {
         "lockDurationAsIso8601": {
-          "description": "The lock duration",
-          "format": "date-time",
+          "description": "The lock duration. Range: 5 Sec - 5 Min.",
           "type": "string"
         },
         "ttlAsIso8601": {
-          "description": "The time to live",
-          "format": "date-time",
+          "description": "The time to live. Range: 1 Min - 2 Days.",
           "type": "string"
         },
         "maxDeliveryCount": {
-          "description": "The max delivery count",
+          "description": "The max delivery count. Range: 1-100.",
           "format": "int32",
-          "type": "integer"
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 100
         }
       }
     },
     "CloudToDeviceProperties": {
-      "description": "Contains user facing view of Iot Hub Cloud-To-Device messaging properties",
+      "description": "The Iot Hub Cloud-To-Device messaging properties.",
       "type": "object",
       "properties": {
         "maxDeliveryCount": {
-          "description": "The max delivery count",
+          "description": "The max delivery count for the device queue. Range : 1-100.",
           "format": "int32",
-          "type": "integer"
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 100
         },
         "defaultTtlAsIso8601": {
-          "description": "The default time to live",
-          "format": "date-time",
+          "description": "The default time to live for the device queue. Range : 1 Min - 2 Days.",
```

Comment:
What does the "Range: 1 Min - 2 Days" mean in this context?
The format date-time has been removed, and the property name still has *AsIso8601 (a date-time format) in its name. It is confusing.